web/src/timerender.ts

```diff
@@ -353,7 +354,7 @@ function render_date_span($elem: JQuery, rendered_time: TimeRender): JQuery {
 // (What's actually spliced into the message template is the contents
 // of this DOM node as HTML, so effectively a copy of the node. That's
 // okay since to update the time later we look up the node by its id.)
-export function render_date(time: Date): JQuery {
+export function render_date(time: Date): HTMLElement {
```

Comment:
Should this be `JQuery<HTMLElement>`? It is a jQuery object still, right?

Reply:
no, doing `$element[0]` on a `JQuery` object (which is by default `JQuery<HTMLElement>` returns `HTMLElement`